### DISCUSSION TITLE
Filter Bias for Large Farms

### DISCUSSION THREAD

**Discussion by Team4N6** [Chia-Network/chia-blockchain#3976] [Q&A]
*Update*
I think I'm just paranoid. After turning up logging and analyzing several hours of attempts, there were some instances with 4 plots past the filter. It's been a while since I had a stats class, but comparing the numbers between the two farms they seem reasonable.

I'm curious what kind of filter pass rate others are seeing. The documentation suggestion 511/512 should be filtered out.

----------------------------

I seem to notice a bias in the filter based on farm size, but maybe there's something else that I don't understand. I currently have two farms. My primary farm has 267 plots. My other farm has 50 plots. As expected, my smaller farm normally has 0 plots that pass the filter, but not uncommonly will have 1, and very occasionally 2 plots that pass. My primary farm has 1 or 2 plots that pass more frequently, and every once in a great while 3, but still 0 the majority of the time.

If the filter was consistent, I would expect a much larger variability in the number of plots that pass the filter. If a small farm of 50 plots can ever have 2 that pass, then a farm 5x the size should have 4+ that pass sometimes. I never see 4, 5, 6, etc. plots pass the filter with my primary farm. Is it better to run 10 farms with 50 plots each than a single 500 plot farm?

**Comment by nomadengineer** [comment]
Hello,
By 2 farms, are they being farmed by a common farmer or are they 2 separate accounts?
I have a multi harvester network, 3 harvesters connected to 1 farmer. I have been asking that whether a challenge is proofed against all plots in all harvesters or each harvester gets a different challenge! no luck yet! :D

**Reply by Team4N6** [reply]
Two separate accounts on completely different networks. Although I turned logging up and there were a couple occasions today with 4 plots past the filter, so maybe I'm just paranoid.
